test/user/user_test.dart

```diff
@@ -21,7 +21,7 @@ Future main() async {
     expect(me.commentKarma, equals(0));
     expect(me.linkKarma, equals(1));
     expect(me.goldCreddits, equals(0));
-    expect(me.created, equals(1501830779.0));
+    expect(me.createdUtc.millisecondsSinceEpoch / 1000, equals(1501830779));
```

Comment:
Why not have:
`expect(me.createdUtc.millisecondsSinceEpoch, equals(1501830779000));`

Reply:
The RHS is the value that's actually returned by the Reddit API, so this is just so that's clear.